content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -62,12 +62,14 @@ _verify_contains snip_verify_traffic_sleep_to_ingress "$snip_verify_traffic_slee
 _verify_contains snip_verify_traffic_sleep_to_productpage "$snip_verify_traffic_sleep_to_productpage_out"
 _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traffic_notsleep_to_productpage_out"
 
-snip_adding_your_application_to_the_ambient_mesh_1
+_verify_same snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
 
 # test traffic after ambient mode is enabled
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_2 "$snip_adding_your_application_to_the_ambient_mesh_2_out"
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_4 "$snip_adding_your_application_to_the_ambient_mesh_4_out"
+snip_adding_your_application_to_the_ambient_mesh_2
+_verify_like snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
```

Comment:
Actually, verify_like won't work here because of token count mismatch. Just change it to explicitly check for the required output for now, and I'll think about how to improve in a followup.
```suggestion
_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "received netns, starting proxy"
```

Reply:
right, that part will work, but not ideal. The test had other problems previously.